docs/tutorial/code/go/task.yaml

```diff
@@ -0,0 +1,110 @@
+###########################################
+# IMPORTANT
+# Comments matter!
+# The docs use the wrapping comments as
+# markers for including said instructions
+# as snippets in the docs.
+###########################################
+summary: Getting started with Go tutorial
+
+environment:
+
+execute: |
+  set -x
+  # [docs:install-go]
+  sudo snap install go --classic
+  # [docs:install-go-end]
+
+  # [docs:mod-init-go]
+  go mod init go-hello-world
+  go mod tidy
```

Comment:
Is `go mod tidy` needed right after a `go mod init`? It shouldn't be as far as I can tell.

Reply:
Yeah, it is not needed. Thanks for the comment!